lib/help/_skills/bloodlust

```diff
@@ -0,0 +1,17 @@
+Bloodlust represents a warrior's love of battle and their desire to
+continue the fight.  With each foe vanquished, the warrior's bloodlust
+continues to grow, increasing their effectiveness in battle.  Moreover,
+the accompanying adrenaline rush can allow the warrior to continue to 
+fight and sustain themselves, well beyond the stamina and vitality of
+the average adventurer.  However, this fervor of battle can fade as quickly
+as it comes on once a battle has ended.
+
+Rumor has it that some tribes of northern Berskerers have further mastered
```

Comment:
Oooh, opportunity to tie this into racial subcategories!

Reply:
We were floating around some ideas for synergies with various races and shapeshifts recently - definitely something to consider in the future, especially if it's not game-changing buffs/advantages but just adds a bit of flavor.